server/src/main/java/org/opensearch/http/AbstractHttpServerTransport.java

```diff
@@ -84,6 +83,7 @@
 import static org.opensearch.http.HttpTransportSettings.SETTING_HTTP_PORT;
 import static org.opensearch.http.HttpTransportSettings.SETTING_HTTP_PUBLISH_HOST;
 import static org.opensearch.http.HttpTransportSettings.SETTING_HTTP_PUBLISH_PORT;
+import static org.opensearch.transport.TcpTransport.resolveTransportPublishPort;
```

Comment:
This is not needed anymore, right?

Reply:
Yep sorry. Neglected to move resolveTransportPublishPort as well.